src/components/Button/SubmitPanel.tsx

```diff
@@ -0,0 +1,88 @@
+import { Box, Link, Paper, Slide } from "@mui/material";
+import { PropsWithChildren, ReactNode, useState } from "react";
+import { useFormikContext } from "formik";
+import {
+  extractErrors,
+  FormikDialogButton,
+  FormikSubmitButton,
+} from "form/FormikButton";
+
+function StatusMessage({
+  errors,
+  isDirty,
+}: {
+  errors: string[];
+  isDirty: boolean;
+}): ReactNode {
+  const [showErrors, setShowErrors] = useState(false);
+
+  if (errors.length === 0) {
+    return isDirty ? "You have unsaved changes" : null;
+  }
+
+  if (errors.length === 1) {
+    return errors[0];
+  }
+
+  return (
+    <Box>
+      <Link underline="hover" onClick={() => setShowErrors((state) => !state)}>
+        You have {errors.length} errors that must be fixed before submitting.
+      </Link>
+      {showErrors && (
+        <ul>
+          {errors.map((v, idx) => (
+            <li key={idx}>{`${v}`}</li>
+          ))}
+        </ul>
+      )}
+    </Box>
+  );
+}
+
+interface Props {
+  isCreate: boolean;
+  useDialog?: boolean;
+  dialogTitle?: string;
+  dialogMessage?: string;
+}
+
+export function SubmitPanel(props: PropsWithChildren<Props>) {
+  const { dirty, errors, submitCount } = useFormikContext();
+  // when creating a new item, we don't want to bombard with a whole load
+  // of validation errors. So wait until it's been submitted at least once
+  // before dumping the set of things that need to be completed.
+  const errorStrings =
+    props.isCreate && submitCount < 1 ? [] : extractErrors(errors);
+
+  console.log(props);
+  return (
+    <Slide in={true} direction="up">
+      <Paper
```

Comment:
I think this is this bar, correct?

<img width="1565" alt="Screenshot 2023-10-11 at 13 43 03" src="https://github.com/brave/ads-ui/assets/51444/556b4b6f-bb65-4a3f-8cc0-e3bc337aac82">

Could you make the corners rounded the same amount as the other boxes?  And I suspect white would look better too in this case, to match.